app/scenes/Document/components/Header.tsx

```diff
@@ -337,4 +343,10 @@ const Status = styled(Action)`
   color: ${(props) => props.theme.slate};
 `;
 
+const Wrapper = styled(Flex)<{ $hideHeader: boolean }>`
+  opacity: ${(props) => (props.$hideHeader ? 0 : 1)};
+  visibility: ${(props) => (props.$hideHeader ? "hidden" : "visible")};
```

Comment:
Intrigued as to why `visibility` was also needed? I'm sure there's a good reason

Reply:
To not trigger the click/hover events.